codec-http/src/main/java/io/netty/handler/codec/http/HttpUtil.java

```diff
@@ -313,6 +317,62 @@ public static void setTransferEncodingChunked(HttpMessage m, boolean chunked) {
         }
     }
 
+    /**
+     * Fetch charset from message's Content-Type header.
+     *
+     * @return the charset from message's Content-Type header or {@link java.nio.charset.StandardCharsets#ISO_8859_1}
```

Comment:
`java.nio.charset.StandardCharsets` only exists in java7. Link to `CharsetUtil` please.